omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb

```diff
@@ -100,6 +100,7 @@
            {db_port, <%=  @postgresql['port'] %>},
            {db_user, "<%= @sql_user %>"},
            {db_name, "bookshelf" },
+           {db_options, [{ssl, <%= { 'disable' => false, 'prefer' => true, 'require' => 'required' }[@postgresql['sslmode']] %>}]},
```

Comment:
We should move this into a helper function I think.

Reply:
```
{db_options, [{ssl, <%= @helper.postgresql_sslmodes[@postgresql[@postgresql['sslmode']] %>}]},
```